CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 1.15.0 (July 16, 2024)
+
+ * [#4283](https://github.com/redhat-developer/vscode-openshift-tools/pull/4283) Added validation for container image to be deployed from an image URL
+ * [#4255](https://github.com/redhat-developer/vscode-openshift-tools/issues/4255) Remove outdated method from the extension activation
+ * [#4247](https://github.com/redhat-developer/vscode-openshift-tools/pull/4247) Remove dependency to 'make-fetch-happen' NPM module
+ * [#4233](https://github.com/redhat-developer/vscode-openshift-tools/issues/4233) Cluster Webview: sandbox detection status never ends
+ * [#4183](https://github.com/redhat-developer/vscode-openshift-tools/issues/4183) ODO CLI binary stopped working after Devfile schema update to v.2.2.2
+ * [#4172](https://github.com/redhat-developer/vscode-openshift-tools/issues/4172) There is no any error/warning notification when Helm Chart cannot be installed
```

Comment:
Here, I would just say : "There is no error/warning notification when Helm Chart cannot be installed"